assignment-1/Players.java

```diff
@@ -0,0 +1,53 @@
+public class Players {
+    private String name;
+    private String team;
+    private String role;
```

Comment:
Best practice would be to store the roles as an enum and use the enum constants wherever required instead of plain strings.